app/models/statement.rb

```diff
@@ -59,6 +59,18 @@ def marked_as_paid?
     marked_as_paid_at.present? && paid?
   end
 
+  def name
```

Comment:
those feel like FE related, maybe belong in a helper/component instead, also could we add a test please?